iot-hub/Samples/device/X509DeviceCertWithChainSample/readme.md

```diff
@@ -0,0 +1,21 @@
+# x509 device certificate with chain sample
+
+## How to run the sample
+
+1) Create IotHub and device to use in sample.
```

Comment:
shouldn't creating a device be after you run the AuthSetup in order to grab the thumbprint that is generated by the script and use it when creating a device?

Reply:
Not needed if you use the x.509 CA signed option. It is simpler to use that option. But both options will work fine. Though you are correct of the order of things if you use the self signed option.